examples/record/sketch.js

```diff
@@ -7,17 +7,14 @@ var mic, recorder, soundFile;
 var state = 0; // mousePress will increment from Record, to Stop, to Play
```

Comment:
I agree with @endurance21 on updating `var` to `let`.
Regarding the `state` variable, since it was originally defined that way I would touch the logic of the example. What I definitely suggest is to make the comment after the definition of `state` more explicit, like:

```js
// mousePress will increment from state = 0 (Record), to state = 1 (Stop), to statue = 2 (Play)
```

Reply:
Yeah,
I've implemented it in the 2nd commit,
if the 2 boolean approach is not merged and we are left with a state variable, it will be retained in the code